### ISSUE TITLE
Oliver thoughts

### ISSUE THREAD

**Issue by CamiloSinningUN** [Smrevilo/ANNDL-Challenges#8]
If this works, it is very important to remember that the training set and validation set both right now are not 100% augmented images (some are left untouched) and the distribution between them are more or less random from my side. Secondly, I have also made a somewhat random (slightly heavier than default) choice on the level of augmentation of each image. These are all things that we need to play around with if it works!
